cardano-api/src/Cardano/Api/Fees.hs

```diff
@@ -1038,8 +1039,8 @@ makeTransactionBodyAutoBalance systemstart history pparams
         createAndValidateTransactionBody finalTxBodyContent
     return (BalancedTxBody finalTxBodyContent txbody3 (TxOut changeaddr balance TxOutDatumNone ReferenceScriptNone) fee)
  where
-   bpparams = bundleProtocolParams era' pparams
-
+   bpparams'e :: Either TxBodyErrorAutoBalance (BundledProtocolParameters era)
+   bpparams'e = first (TxBodyError . TxBodyProtocolParamsConversionError) $ bundleProtocolParams era' pparams
```

Comment:
Using special characters (in this case the apostrophe) in variable names harder to read and understand. I was actually confused at first as I thought `'` was some kind of operator.

Reply:
That's a one way of encoding that the variable in scope is similar to the other one, but it's wrapped in either. I've changed to prefix `e` to match the convention how we mark `Maybe`s.